### PR TITLE
Deploying COS to demo

### PR DESCRIPTION
**Pull request by jvictorcf**
### Change description ###
Deploying COS to Demo

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: k8s/demo/common/divorce/div-cos.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: div-cos
+    version: 1.3.2
+  values:
+    java:
+      replicas: 2
+      memoryRequests: '768Mi'
```
Comment (timja) [inline k8s/demo/common/divorce/div-cos.yaml]: you shouldn't need these memory limited and cpu requests

Comment (jvictorcf) [reply k8s/demo/common/divorce/div-cos.yaml]: Why? Are these the default values?
Should I remove it only from the demo file or from all of them?

File: k8s/demo/common/divorce/div-cos.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      cpuRequests: '1000m'
+      memoryLimits: '3072Mi'
+      cpuLimits: '2500m'
+      readinessDelay: 45
+      readinessTimeout: 5
+      readinessPeriod: 15
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "63c30ef9-a27b-4d38-8afc-4c180fb6ac33"
```
Comment (timja) [inline k8s/demo/common/divorce/div-cos.yaml]: is this the demo key?

File: k8s/demo/common/divorce/div-cos.yaml (hunk 1 above)
Comment (timja) [reply k8s/demo/common/divorce/div-cos.yaml]: tbh, not sure the history, 95% of apps don't need it, it was probably copied from the original, but might be too much work to find out

File: k8s/demo/common/divorce/div-cos.yaml (hunk 2 above)
Comment (jvictorcf) [reply k8s/demo/common/divorce/div-cos.yaml]: Good question. Let me find out.

Comment (jvictorcf) [reply k8s/demo/common/divorce/div-cos.yaml]: It wasn't. Good catch.
Changed it now.

File: k8s/demo/common/divorce/div-cos.yaml (hunk 1 above)
Comment (jvictorcf) [reply k8s/demo/common/divorce/div-cos.yaml]: fair enough. I'll leave it, in that case.

**Pull request merged**